tests/simulation/test_raspa2.py

```diff
@@ -0,0 +1,26 @@
+from mofa.simulation.raspa2 import RASPA2Runner
+from pytest import mark
+
+
+@mark.skip()
```

Comment:
How long does it take to run? If under a minute, could we run this as part of the normal test suite?